src/t_list.c

```diff
@@ -921,18 +927,22 @@ void rpoplpushCommand(client *c) {
  * 3) Propagate the resulting BRPOP, BLPOP, BLMPOP and additional xPUSH if any into
  *    the AOF and replication channel.
  *
+ * The argument 'deleted' indicates whether the list is deleted.
+ *
  * The argument 'wherefrom' is LIST_TAIL or LIST_HEAD, and indicates if the
  * 'value' element was popped from the head (BLPOP) or tail (BRPOP) so that
  * we can propagate the command properly.
  *
  * The argument 'whereto' is LIST_TAIL or LIST_HEAD, and indicates if the
  * 'value' element is to be pushed to the head or tail so that we can
  * propagate the command properly. */
-void serveClientBlockedOnList(client *receiver, robj *o, robj *key, robj *dstkey, redisDb *db, int wherefrom, int whereto)
+void serveClientBlockedOnList(client *receiver, robj *o, robj *key, robj *dstkey, redisDb *db, int *deleted, int wherefrom, int whereto)
```

Comment:
output variables are usually put last, in this case the function returns void, so in theory there's no real need for an output variable, but i guess it makes it more explicit (giving it a name).